ouroboros-consensus/src/Ouroboros/Consensus/Protocol/Abstract.hs

```diff
@@ -50,9 +50,13 @@ import qualified Ouroboros.Network.AnchoredFragment as AF
 import           Ouroboros.Network.Block (HasHeader (..), SlotNo (..))
 import           Ouroboros.Network.Chain (Chain)
 
+-- TODO Better place to put the Empty class?
```

Comment:
_If_ we still need it, let's move it to `Util` (but with a bit of luck it'll be gone).